Halftime/McGriddles/src/js/main.js

```diff
@@ -0,0 +1,14 @@
+/* tracer bullet */
+$(document).ready(function() {
```

Comment:
This is actually a CSS assignment - great to see you were starting to think about jQuery implementation.